api/src/main/java/com/datastrato/gravitino/SupportsCatalogs.java

```diff
@@ -2,12 +2,8 @@
  * Copyright 2024 Datastrato Pvt Ltd.
  * This software is licensed under the Apache License version 2.
  */
-package com.datastrato.gravitino.rel;
```

Comment:
These two package refactoring will be conflict with what I did in the PR https://github.com/datastrato/gravitino/pull/3419

Reply:
Yeah, I will rebase this PR when that one is merged.